src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,116 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Task;
+
+/**
+ * Handles actions by user with regards to all their tasks.
+ */
+public class TaskList {
+    private Storage savedStorage;
+    private List<Task> allItems;
+
+    /**
+     * Creates an empty TaskList.
+     */
+    public TaskList() {
+        this.savedStorage = new Storage();
+        this.allItems = this.savedStorage.getSavedTasks();
+    }
+
+    /**
+     * Adds task to the current list of tasks.
+     *
+     * @param toAdd Task to add
+     */
+    public void addTask(Task toAdd) {
+        this.allItems.add(toAdd);
+        String printTask = "Alright, its in your list now!\n\t" + toAdd
+                + "\nNow you have " + this.allItems.size() + " tasks.\n" + Ui.LINE;
+        Ui.printMessageToUser(printTask);
+    }
+
+    /**
+     * Retrieves all the tasks currently stored.
+     *
+     * @return the tasks currently stored in a List
+     */
+    public List<Task> getAllTasks() {
+        return this.allItems;
+    }
+
+    /**
+     * Prints the output of all tasks in store currently.
+     */
+    public void printStore() {
+        String printList;
+        if (this.allItems.size() == 0) {
+            printList = "There are no tasks added till now.\nAdd one by just typing its name.\n"
+                    + Ui.LINE;
+        } else {
+            printList = "Please finish these tasks ASAP!\n";
+            int counter = 1;
+            for (Task task : this.allItems) {
+                printList = printList.concat("[" + counter + "] " + task + "\n");
+                counter++;
+            }
+            printList = printList.concat("If you're brave enough to start,\n"
```

Comment:
hahaha i like the way you make it sounds so engaging!

Reply:
Hahaha thanks!